File: php/PROBLEMS/1116.php
<?php
$n = intval(fgets(STDIN));

for($i=0;$i<$n;$i++){
    $e = fgets(STDIN);
    $e = explode(' ', $e);
    $x = intval($e[0]);
    $y = intval($e[1]);

    if ($y!=0){
        printf("%.1f\n", $x/$y);
    }else{
        printf("divisao impossivel\n");
    }
}

?>

File: c/PROBLEMS/1116.c
#include <stdio.h>

int main(){
    float x, y, n, i;

    scanf("%f", &n);

    for(i=0;i<n;i++){
        scanf("%f %f", &x, &y);

        if (y!=0){
            printf("%.1f\n", x/y);
        }else{
            printf("divisao impossivel\n");
        }
    }
}